source/Depreciation.cpp

```diff
@@ -63,6 +63,41 @@ double Depreciation::Full()
 
 
 
+// Calculate the value fraction for an item of the given age.
+double Depreciation::Depreciate(int age)
```

Comment:
Is there a problem with making it public?

Reply:
I cant see an advantage to making it public.
It being private helps seeing which functions one can call for this class and even in the class itself separate internal business' neat.
Also in general good practices are to make stuff private if it can be, I think most of the project fits that?

Reply:
Advantages: not having errors.
Apparently a different line in a different file uses it, so I don't think this can happen. Is that alright?